common-npm-packages/ios-signing-common/Tests/L0InstallCertInTemporaryKeychain.ts

```diff
@@ -27,14 +27,16 @@ const stdOuts = [
     { keychainPath: 'keychainPath2', keychainPwd: 'keychainPwd2', p12CertPath: 'p12CertPath2', p12Pwd: 'p12Pwd2', useKeychainIfExists: false, skipPartitionIdAclSetup: true },
     { keychainPath: 'keychainPath3', keychainPwd: 'keychainPwd3', p12CertPath: 'p12CertPath3', p12Pwd: 'p12Pwd3', useKeychainIfExists: true, skipPartitionIdAclSetup: true },
     { keychainPath: 'keychainPath4', keychainPwd: 'keychainPwd4', p12CertPath: 'p12CertPath4', p12Pwd: 'p12Pwd4', useKeychainIfExists: true, skipPartitionIdAclSetup: true },
-    { keychainPath: 'keychainPath5', keychainPwd: 'keychainPwd5', p12CertPath: 'p12CertPath5', p12Pwd: 'p12Pwd5', useKeychainIfExists: true, skipPartitionIdAclSetup: false },
-    { keychainPath: 'keychainPath6', keychainPwd: 'keychainPwd6', p12CertPath: 'p12CertPath6', p12Pwd: 'p12Pwd6', useKeychainIfExists: true, skipPartitionIdAclSetup: false },
+    { keychainPath: 'keychainPath5', keychainPwd: 'keychainPwd5', p12CertPath: 'p12CertPath5', p12Pwd: 'p12Pwd5', useKeychainIfExists: true, skipPartitionIdAclSetup: false},
```

Comment:
Nit: the space was deleted, probably unintentionally

Reply:
Fixed, thanks!